### ISSUE TITLE
Why I can't see subnetMask in Device ?

### ISSUE THREAD

**Issue by DenisMashintsev** [mavris/MMLanScan#25]
In current version property subnetMask in Device object always == nil.

**Comment by jpalten** [comment]
It might take a few secs to acquire. Does it stay nil, or will it correct after a while?

**Comment by DenisMashintsev** [comment]
Yes, it stays nil. The problem is in the getMACDetails method of the MACOperation class. In this method, the subnetMask property of the Device object is not initialized.

**Comment by mavris** [comment]
I will check this out and I will let you know soon.
